backend/app/model/index/Page.scala

```diff
@@ -18,16 +18,22 @@ object PageDimensions {
 
 case class HighlightSpan(x: Double, y: Double, width: Double, height: Double, rotation: Double)
 
-sealed abstract class PageHighlight { def id: String }
-case class SearchHighlight(id: String, spans: List[HighlightSpan]) extends PageHighlight
-case class FindHighlight(id: String, spans: List[HighlightSpan]) extends PageHighlight
+sealed abstract class PageHighlight {
+  def id: String
+  def index: Int
+  def spans: List[HighlightSpan]
+}
+// TODO: these are identical... why do we need them?
```

Comment:
I think the idea was the `type` that gets inserted when serialized can be used to style them differently. This type existed before I did my work on `viewer2` so I'm not actually sure :)